docs/source/Plugin/P014.rst

```diff
@@ -57,9 +57,9 @@ Do not use the single ended circuit as the Vout is connected directly to ground:
 
 
 **Note:**
-The plugin reads the CHIP ID to determine if the ADC is available.
+The plugin reads the CHIP ID to determine if the ADC is available. This Plugin is NOT compatible with STH20, for SHT2x devices use Plugin 122.
 
-SHT20 and HTU21D devices while using the same reading protocol do not have a CHIP ID capability and not able to detect them.
+HTU21D devices while using the same reading protocol do not have a CHIP ID capability and not able to detect them.

```

Comment:
This should also list SI70xx sensors, as those are also supported by this plugin.

Reply:
so also "SI70xx sensors" don't have CHIP ID capability?

Reply:
Ah, I misread that line, but I'm not sure if that statement is actually true for the HTU21D, as it _is_ being detected in some way...